server/src/main/java/org/apache/uniffle/server/storage/HadoopStorageManager.java

```diff
@@ -99,6 +138,11 @@ public Storage selectStorage(ShuffleDataReadEvent event) {
 
   @Override
   public void removeResources(PurgeEvent event) {
+    removeResources(event, false);
+  }
+
+  @Override
+  public void removeResources(PurgeEvent event, boolean isQuick) {
```

Comment:
Could we make the `softDeletion/isQuick` as the internal variable in the PurgeEvent?